src/types.ts

```diff
@@ -20,18 +20,33 @@ export interface RTDBMetadata {
 }
 
 export interface Config {
-  project: 'testnet' | 'mainnet'
+  project: Environment
   databaseUrl: string
 }
 
-export interface RawTokenInfo {
-  address: string
-  decimals: number
+export interface TokenInfo {
   name: string
   symbol: string
+  decimals: number
+  tokenId: string
+  address?: string
   imageUrl?: string
+  isNative?: boolean
   isCoreToken?: boolean
   pegTo?: string
+  isSupercharged?: boolean
+}
+
+export enum Network { // environment-agnostic network name
+  celo = 'celo',
+  ethereum = 'ethereum',
+}
+
+export enum NetworkId { // environment-specific
+  ['celo-mainnet'] = 'celo-mainnet',
+  ['celo-alfajores'] = 'celo-alfajores',
+  ['ethereum-mainnet'] = 'ethereum-mainnet',
+  ['ethereum-sepolia'] = 'ethereum-sepolia',
 }
```

Comment:
I prefer the enums because they allow you to reference values like `Network.celo` instead of typing out "celo" each time, and my IDE does autocomplete for me when I do that. Though I agree that as far as type safety goes, they're equivalent

Reply:
Ah it's an editor thing.
VS code autocompletes just fine without enums.
Not the case with jetbrains?